js/components/Compass/index.js

```diff
@@ -1,8 +1,10 @@
+/* global require */
```

Comment:
Why is this needed? If we support `require()` syntax can't we define that globally in the app's `.eslintrc`?